src/core/mount.c

```diff
@@ -54,6 +54,16 @@ static int mount_dispatch_io(sd_event_source *source, int fd, uint32_t revents,
 static void mount_update_unit_state(Mount *m, Set *gone, Set *around);
 static int mount_process_proc_self_mountinfo(Manager *m);
 
+struct mount_unit_hash {
+        Hashmap *by_mnt_id;
+        Hashmap *by_device;
+};
+
+static struct mount_unit_hash mount_hash = {
```

Comment:
No static variables please. Please place this in the `Manager` structure, i.e. similar to how `swaps_by_devnode() exists`